### PR TITLE
Make schema consistent and update docs

### PR DESCRIPTION
**Pull request by j-freddy**
This small PR updates `mutation.graphql` to be consistent with `createPlayer.ts` which expects a payload argument. `SETUP_ROBUST.md` is updated so the examples work with the latest schema definitions.

NI: @professor-l I do not know your playstyle and just set it to DAS.

### CODE REVIEW

**Review by professor-l — APPROVED** [review]
LGTM, thanks for this! All my competitive games from years past are TAP, but I only really play DAS nowadays so we can leave it :P.

 Out of curiousity, do you have merge perms once I approve this? If not, do you want them? How much contributing are you planning on doing?

**Comment by j-freddy** [comment]
I don't have merge perms. Up to you.

I don't know how much I'll be able to contribute. I have very little experience in databases. also have little time after Easter, so I'll just be hovering around when I'm free.

**Pull request merged**
